### PR TITLE
feat: support WebAssembly runtime

### PR DESCRIPTION
**Pull request by wujunzhuo**

### CODE REVIEW

File: cmd/run.go [hunk 1]
```diff
@@ -41,6 +42,7 @@ var runCmd = &cobra.Command{
 		loadViperValue(cmd, runViper, &opts.Name, "name")
 		loadViperValue(cmd, runViper, &opts.ModFile, "modfile")
 		loadViperValue(cmd, runViper, &opts.Credential, "credential")
+		loadViperValue(cmd, runViper, &opts.WasmRuntime, "wasm-runtime")
```
Comment (fanweixiao) [inline cmd/run.go:45]: If `Runtime` is better than `WasmRuntime` when considering deno support in the future?

File: example/wasm/rust/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "app"
```
Comment (fanweixiao) [inline example/wasm/rust/Cargo.toml:2]: if `sfn` is better?

**Review by fanweixiao — APPROVED** [review]

File: cmd/run.go (hunk 1 above)
Comment (wujunzhuo) [reply cmd/run.go:45]: I prefer to compile (bundle) js program with `.deno` suffix, and regeister with `DenoRuntime`

File: example/wasm/rust/Cargo.toml (hunk 2 above)
Comment (wujunzhuo) [reply example/wasm/rust/Cargo.toml:2]: 👌

File: cmd/run.go (hunk 1 above)
Comment (fanweixiao) [reply cmd/run.go:45]: @wujunzhuo I think `.ts` with deno engine is much more convenient.

**Pull request merged**